google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/LocalConnectionCheckerTest.java

```diff
@@ -63,4 +74,23 @@ public void localConnectionCheckerWorksWithMockSpanner() {
       while (resultSet.next()) {}
     }
   }
+
+  @Test
+  public void testNoRunningEmulator() {
+    final String uri =
+        "cloudspanner://localhost:42424/projects/proj/instances/inst/databases/db?usePlainText=true";
```

Comment:
nit: It might be safer to use `server.getPort() - 1` instead of `42424`, as there is a 1/65000 chance that the port assigned to the mock server in this test happens to be `42424`.

Reply:
Good point, updated it.